index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+	<title>Apple</title>
+	<link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.8.1/css/all.css"
+      integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf"
+      crossorigin="anonymous"/>
+	<link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+	<main>
+		<div id="container">
+			<div id="main-content">
+				<nav id="nav-bar">
+					<ul id="menu-list">
+					<!-- The comment tags below is use to remove inline-block space -->
+						<li class="menu-button"><button><i class="fab fa-apple"></i></button></li><!-- 
+						 --><li class="menu-button"><button>Store</button></li><!-- 
+						  --><li class="menu-button"><button>Mac</button></li><!-- 
+						  --><li class="menu-button"><button>iPod</button></li><!-- 
+						  --><li class="menu-button"><button>iPhone</button></li><!-- 
+						  --><li class="menu-button"><button>iPad</button></li><!-- 
+						  --><li class="menu-button"><button>iTunes</button></li><!-- 
+						  --><li class="menu-button"><button>Support</button></li><!-- 
+						  --><li class="menu-button search-input"><input type="text" name="search"><i class="fas fa-search"></i></li>
+					</ul>
+				</nav>
+				<div id="main-links">
+					<div id="first-link" class="link">
+						<a href="#"> <img src="images/first.png" alt="main-link-image"></a>
```

Comment:
`SUGGESTION`

Since this image is a text, consider adding more descriptive `alt`, for example, the text equivalent of the image itself.